forch/forchestrator.py

```diff
@@ -297,22 +299,33 @@ def _calculate_config_files(self):
         if gauge_config_file:
             self._gauge_config_file = os.path.join(self._forch_config_dir, gauge_config_file)
 
-        segments_vlans_file = orch_config.segments_vlans_file
-        if segments_vlans_file:
-            self._segments_vlans_file = os.path.join(self._forch_config_dir, segments_vlans_file)
-
         structural_config_file = orch_config.structural_config_file
-        if structural_config_file:
-            self._structural_config_file = os.path.join(
-                self._forch_config_dir, structural_config_file)
-
-            if not os.path.exists(self._structural_config_file):
-                raise Exception(
-                    f'Structural config file does not exist: {self._structural_config_file}')
+        if not structural_config_file:
+            return False
 
-            return True
+        self._structural_config_file = os.path.join(
+            self._forch_config_dir, structural_config_file)
 
-        return False
+        if not os.path.exists(self._structural_config_file):
+            raise Exception(
+                f'Structural config file does not exist: {self._structural_config_file}')
+
+        self._segments_vlans_file = os.path.join(
+            self._forch_config_dir, orch_config.segments_vlans_file)
+        if not os.path.exists(self._segments_vlans_file):
+            error_msg = f'Segments-to-vlans file does not exist: {self._segments_vlans_file}.'
+            error_msg += ' DVA disabled'
```

Comment:
would prefer a single assignment rather than appending a string.  You can do this with % notation and ()'s or something if the line gets too long.

Reply:
updated. PTAL